website/api/controllers/webhooks/receive-usage-analytics.js

```diff
@@ -8,9 +8,18 @@ module.exports = {
 
 
   inputs: {
-    anonymousIdentifier: { required: true, type: 'string', example: '1', description: 'An anonymous identifier telling us which Fleet deployment this is.', },
+    anonymousIdentifier: { required: true, type: 'string', example: '9pnzNmrES3mQG66UQtd29cYTiX2+fZ4CYxDvh495720=', description: 'An anonymous identifier telling us which Fleet deployment this is.', },
     fleetVersion: { required: true, type: 'string', example: 'x.x.x' },
+    licenseTier: { required: true, type: 'string', example: 'free' },
```

Comment:
you might consider adding an isIn validation here as a failsafe (that way it'll cause a 500 if that ever changes, which will alert and notify the team that something unexpected changed) https://sailsjs.com/documentation/concepts/models-and-orm/validations#?validation-rules